### DISCUSSION TITLE
Download Folder Structure, option to customize it?

### DISCUSSION THREAD

**Discussion by NitsuSaiNeko** [bitbybyte/fantiadl#81] [Q&A] [answered]
Hi, can you list what is the current download directory folder structure and output format? 

Also, is there a way to customize it? If not, any plan to add support for custom output folder structure/format?

Thank you!

**Comment by bitbybyte** [comment, marked as answer]
Hello, please see #31. While I agree it would be nice to have custom download directories, this is unfortunately difficult to achieve given the nature of Fantia posts and the structure of the program.

The general directory structure is:

```/{fanclub_creator_name}
/{fanclub_creator_name}/{post_id}
/{fanclub_creator_name}/{post_id}/metadata.json
/{fanclub_creator_name}/{post_id}/.incomplete
/{fanclub_creator_name}/{post_id}/thumb.[jpg|png|...]
/{fanclub_creator_name}/{post_id}/{post_content_title}
/{fanclub_creator_name}/{post_id}/{post_content_title}/...
```

Each fanclub post has multiple post contents, which usually represent the parts of posts available to each fanclub plan. Each has its own directory inside of the root `/{fanclub_creator_name}/{post_id}` using the title of the post content. Photo galleries by default will be numbered starting at 0. You can use `-s` to download files using their original server filenames across the program. Additional files including headers and icons will be written to `/{fanclub_creator_name}` if you use the `-m` flag.
